netbox/extras/views.py

```diff
@@ -1343,11 +1163,21 @@ def get(self, request, job_pk):
         module = job.object
         script = module.scripts[job.name]()
 
+        legacy_script = False
+        legacy_report = False
+        if job.data and ('logs' not in job.data):
+            if 'log' in job.data:
+                legacy_script = True
+            else:
+                legacy_report = True
```

Comment:
I'm not clear on the concept of a legacy script: We should avoid introducing this distinction if at all possible.

Reply:
Added in the summary (success, failure, error) of reports to scripts as part of the merge, the old scripts data is in a truncated format so isn't compatible without doing this.